lib/packagekit-glib2/pk-progress-bar.c

```diff
@@ -268,8 +268,9 @@ pk_progress_bar_start (PkProgressBar *self, const gchar *text)
 	g_return_val_if_fail (PK_IS_PROGRESS_BAR (self), FALSE);
 
 	/* same as last time */
-	if (g_strcmp0 (self->priv->old_start_text, text) == 0)
-		return TRUE;
+	if (self->priv->old_start_text != NULL && text != NULL)
+		if (g_strcmp0 (self->priv->old_start_text, text) == 0)
```

Comment:
Can the "outer" `if` please have `{}` to make it clear please. The inner one is fine as it is.